pkg/update/update.go

```diff
@@ -70,6 +70,42 @@ func (c *updater) WithStatusWriter(writer statuswriter.StatusWriter) *updater {
 	return c
 }
 
+func (c *updater) VersionUpdate(ctx context.Context, pkgName string, pkgVersion string) (*UpdateTransaction, error) {
```

Comment:
please call it something similar to `PrepareForVersion`, so that one sees it's doing something similar as the `Prepare` function

Reply:
for `PrepareForVersion` it is showing the lint error of line exceeded

Reply:
but then you can reformat the line to be in multiple lines, for example

```
func (c *updater) PrepareForVersion(
  ctx context.Context,
...
) (...) {
```